pkg/types/powervs/platform.go

```diff
@@ -74,6 +74,11 @@ type Platform struct {
 	// @TODO: make this +optional when we have TF support
 	ImageName string `json:"imageName,omitempty"`
 
+	// Region where the bucket holding the boot image is. If empty the
+	// VPC region is used instead.
+	// +optional
+	ImageBucketRegion string `json:"imageBucketRegion,omitempty"`
```

Comment:
just to keep things simple, can we omit this?

Reply:
Yep can omit.

I used it so I could use the us-east bucket and the eu-gb vpc region but I don't think it's something a customer would be doing.

Reply:
yah, if a customer wouldn't, we should keep it out. otherwise just another thing for us to test since it's available

Reply:
Removed